2022/08-03_kubeflow/pipelines/sample.py

```diff
@@ -0,0 +1,27 @@
+import kfp
+import kfp.components as comp
+
```

Comment:
Add logic to resolve + inject hash for local Docker image when using local component.